src/test/java/org/opentripplanner/transit/model/_data/TransitModelForTest.java

```diff
@@ -52,6 +57,18 @@ public class TransitModelForTest {
   public static final String OTHER_TIME_ZONE_ID = "America/Los_Angeles";
   public static final WgsCoordinate ANY_COORDINATE = new WgsCoordinate(60.0, 10.0);
 
+  // This is used to create valid objects - do not use it for verification
+  private static final Polygon ANY_POLYGON = GeometryUtils
```

Comment:
I created https://github.com/opentripplanner/OpenTripPlanner/blob/90b15cd8684da65bb5bf2e1d984d00bc53129cdc/src/test/java/org/opentripplanner/_support/geometry/Polygons.java#L10 for this. Is it worth moving this constant?

Reply:
I have noticed - there are pro/cos. I can change it to use one of the items in Polygons, i did not because that would violate some of the basic test principles - but pragmatically speaking it would be ok. The problem is that they are public and used elsewhere - so then this is no longer isolated and protected. They can (theoretically) be changed by other tests, and since they are public someone may start to add assertions on them - witch I do not want. On the other hand, this solution creates a bit of unecessary code witch obstruct the reading. I think I will do it next time around, and then at the same time meke the `Polygons` safer by creating the polygons for each test, not reusing them (they are not mutable).

Reply:
Consider this - this is legal: `OSLO.getCoordinates()[0].x = 0.0;`